client/src/components/AddPollDialog.js

```diff
@@ -200,7 +203,6 @@ class AddPollDialog extends Component {
             <div>
                 <Button
                     aria-label="create poll button"
-                    className={classes.button}
```

Comment:
Can you please remove button styles from styles object as they are redundant now